ReleaseNotes-2.10.11

```diff
@@ -0,0 +1,18 @@
+#
+# Open Build Service 2.10.11 (TO BE RELEASED)
+#
+
+Please read the README.md file for initial installation
+instructions or use the OBS Appliance from
+
+  http://openbuildservice.org/download/
+
+The dist/README.UPDATERS file has information for people updating
+from a previous OBS release.
+
+
+Bugfixes
+========
+
+* Frontend:
+  - Update rails to replace mimemagic with marcel (mimemagic was yanked). Details https://weblog.rubyonrails.org/2021/3/26/marcel-upgrade-releases/
```

Comment:
Release notes are for users/admins, not for developers

Reply:
Please have another look